blsftcosi/protocol/sub_protocol.go

```diff
@@ -0,0 +1,431 @@
+package protocol
+
+import (
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/dedis/kyber"
+	"github.com/dedis/kyber/pairing"
+	"github.com/dedis/kyber/sign/bls"
+	"github.com/dedis/kyber/sign/cosi"
```

Comment:
i see that we need to use the mask so we need to import cosi here. but is the cosi mask independent from cosi? if that's the case we probably should move it into its own package like `kyber/sign/mask`?

Reply:
That would make sense yes, the mask should work for any kind of public keys